src/test/java/org/openrewrite/java/testing/assertj/UseExplicitContainsTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2021 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.testing.assertj;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+class UseExplicitContainsTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec
+          .parser(JavaParser.fromJavaVersion().classpath("junit"))
+          .recipe(new UseExplicitContains());
+    }
+
+    @Test
+    void singleStaticMethodNoMessage() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+				import java.util.Collection;
+				import java.util.ArrayList;
+	
+	            import org.junit.jupiter.api.Test;        
+	
+	            import static org.assertj.core.api.Assertions.assertThat;
+	
+	            public class MyTest {
+	
+	                @Test
+	                public void test() {
+	                	Collection collection = new ArrayList();
+	                	collection.add("3");
+	                    assertThat(collection.contains("3")).isTrue();
+	                }
+	            }
+              """,
+            """
+				import java.util.Collection;
+				import java.util.ArrayList;
+	
+	            import org.junit.jupiter.api.Test;       
+	
+	            import static org.assertj.core.api.Assertions.assertThat;
+	
+	            public class MyTest {
+	
+	                @Test
+	                public void test() {
+	                	Collection collection = new ArrayList();
+	                	collection.add("3");
+	                    assertThat(collection).contains("3");
+	                }
+	            }
+              """
+          )
+        );
+    }
+
```

Comment:
Would be good to also add a test using that should not be changed, while still being close to what the recipe matches, so something like: `assertThat(...).isTrue()` where `...` is still a method invocation, but not one to `contains`.

Reply:
Done. I have added a test for isFalse/contains -> doesNotContain, and a test to show that it only affects contains/isTrue and contains/isFalse